tests/results/TextPositionTest.py

```diff
@@ -36,23 +36,54 @@ def test_comparison(self):
         uut2 = TextPosition(None, None)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 > uut2)
+        self.assertFalse(uut1 < uut2)
 
         uut1 = TextPosition(1, 2)
         uut2 = TextPosition(1, 3)
+        uut3 = TextPosition(1, None)
+        uut4 = TextPosition(1, 5)
         self.assertFalse(uut1 >= uut2)
+        self.assertTrue(uut2 > uut1)
+        self.assertFalse(uut3 > uut4)
         self.assertTrue(uut1 <= uut2)
+        self.assertTrue(uut1 < uut2)
+        self.assertFalse(uut1 > uut2)
 
         uut1 = TextPosition(2, None)
         uut2 = TextPosition(3, None)
         self.assertFalse(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 > uut2)
+        self.assertTrue(uut1 < uut2)
 
         uut1 = TextPosition(None, None)
         uut2 = TextPosition(4, 5)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 < uut2)
+        self.assertFalse(uut1 > uut2)
 
         uut1 = TextPosition(4, 8)
         uut2 = TextPosition(4, 8)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut2 <= uut1)
+        self.assertFalse(uut1 > uut2)
+        self.assertFalse(uut2 < uut1)
+
+        uut1 = TextPosition(1, 2)
+        uut2 = TextPosition(4, 3)
+        self.assertTrue(uut1 < uut2)
+        self.assertFalse(uut1 > uut2)
```

Comment:
@NiklasMM I love you ;P for defending `pytest` just like me :) But it seems we have no chance here! Also because people are interested in using **green** as alternative test runner: https://github.com/coala/coala/issues/4060

So join my aspects team and support the separate **coala-aspects** repo via https://github.com/coala/coala/issues/4038 . Then we can do everything as pytestastic as possible there :)